src/assets/js/archive.js

```diff
@@ -5,62 +5,109 @@ const releasesToShow = 99999;
 function fetchFlutterReleases(os, callback, errorCallback) {
   // OS: windows, macos, linux
   const url = `https://storage.googleapis.com/flutter_infra_release/releases/releases_${os}.json`;
-  $.ajax({
-    type: 'GET',
-    url: url,
-    dataType: 'json',
-    success: function (data) { callback(data, os); },
-    error: function (xhr, textStatus, errorThrown) {
-      if (errorCallback) errorCallback(os);
-    }
-  })
+  fetch(url, { type: 'GET'})
+    .then(response => response.json())
+    .then(data => callback(data, os))
+    .catch(_ => { if (errorCallback) errorCallback(os) })
 }
 
 function updateTable(releases, os) {
   const releaseData = releases.releases;
 
   for (const channel in releases.current_release) {
-    const table = $(`#downloads-${os}-${channel}`);
-    table.addClass('collapsed').find('.loading').remove();
+    const table = document.getElementById(`downloads-${os}-${channel}`);
+
+    //table is not present when channel is dev
+    if(!table) {
+      continue;
+    }
 
-    const releasesForChannel = releaseData.filter(function (release) {
+    table.classList.add('collapsed');
+    const loadingElements = table.querySelectorAll('.loading');
+    loadingElements.forEach(function(element) {
+      element.remove();
+    });
+
+    const releasesForChannel = releaseData.filter(function(release) {
       return release.channel === channel;
     });
 
-    releasesForChannel.forEach(function (release, index) {
+    releasesForChannel.forEach(function(release, index) {
       // If this is the first row after the cut-off, insert the "Show more..." link.
       if (index === releasesToShow) {
-        const showAll = $('<a />').text('Show all...').attr('href', '#').click(function (event) {
-          $(this).closest('table').removeClass('collapsed');
-          $(this).closest('tr').remove();
+        const showAll = document.createElement('a');
+        showAll.textContent = 'Show all...';
+        showAll.href = '#';
+        showAll.addEventListener('click', function(event) {
+          this.closest('table').classList.remove('collapsed');
+          this.closest('tr').remove();
           event.preventDefault();
         });
-        $('<tr>').append($('<td colspan="6"></td></tr>').append(showAll)).appendTo(table);
+
+        const row = document.createElement('tr');
+        const cell = document.createElement('td');
+        cell.colSpan = 6;
+        cell.appendChild(showAll);
+        row.appendChild(cell);
+        table.appendChild(row);
```

Comment:
Sure this is a bit longer, but it's more explicit and much easier to understand what it's doing. Cool!